packages/chakra-ui/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -0,0 +1,76 @@
+import { CheckboxGroup, Checkbox, FormLabel } from "@chakra-ui/react";
+import { WidgetProps } from "@rjsf/core";
+import React from "react";
+
+// const selectValue = (value, selected, all) => {
+//   const at = all.indexOf(value);
+//   const updated = selected.slice(0, at).concat(value, selected.slice(at));
+
+//   // As inserting values at predefined index positions doesn't work with empty
+//   // arrays, we need to reorder the updated selection to match the initial order
+//   return updated.sort((a, b) => all.indexOf(a) > all.indexOf(b));
+// };
+
+// const deselectValue = (value, selected) => {
+//   return selected.filter((v) => v !== value);
+// };
+
+const CheckboxesWidget = ({
+  id,
+  disabled,
+  options,
+  value,
+  readonly,
+  onChange,
+  onBlur,
+  onFocus,
+}: WidgetProps) => {
+  const { enumOptions, enumDisabled } = options;
```

Comment:
On the "Arrays" tab -- by default, it looks like the checkbox is showing up as inline:

![image](https://user-images.githubusercontent.com/1689183/111927148-17886e80-8a86-11eb-815c-c2f9afa23856.png)


We should instead show it as having a new line for each item, by default, as with the default theme:
![image](https://user-images.githubusercontent.com/1689183/111927166-240cc700-8a86-11eb-8b9a-1eaec866c4b9.png)

The user should be able to make the checkboxes inline by setting `inline` in ui:options to true (see https://react-jsonschema-form.readthedocs.io/en/latest/usage/arrays/#inline-checkboxes).